src/main/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionController.java

```diff
@@ -118,4 +126,16 @@ public ResponseEntity patchObjection(
             );
         }
     }
+
+    @PostMapping("/{objectionId}/attachments")
+    public ResponseEntity<String> uploadAttachmentToRequest(
+            @RequestParam("file") MultipartFile file, @PathVariable String objectionId) {
+        try {
+            ServiceResult<String> result = objectionService.addAttachment(objectionId, file);
```

Comment:
You seem to be using the objectionId as the requestId.
The objectionId is the id of the objection object in the mongo database
The request Id is generated for each request by eric. We want to be using the requestId from eric as the context argument in all the log statements.
Though It is definitely worth adding the objectionId to the dataMap for the log statement for more information.

If you look at the patchObjection method it shows how to use the data map in the log, and how to get the requestId easily

Reply:
Using data map logging: 

2020-07-08T14:42:28.591Z [8waDSJX3Q2g-2NZM2qQ211SeYzZ5] info: POST /{objectionId}/attachments request received

ZM2qQ211SeYzZ5] info: Finished POST /{objectionId}/attachments request